images/kubekins-e2e/variants.yaml

```diff
@@ -1,32 +1,32 @@
 variants:
   experimental:
     CONFIG: experimental
-    GO_VERSION: 1.12.1
-    K8S_RELEASE: stable
+    GO_VERSION: 1.12.7
+    K8S_RELEASE: latest
```

Comment:
I'm wary of this change, I think the intent is for `experimental` to match `master` in all dimensions except `BAZEL_VERSION`

Looking to see how experimental is used...

Reply:
/cc @BenTheElder @krzyzacy 
I'm also unclear on whether K8S_RELEASE as `stable` (ie: currently `v1.15.2`) is correct for the `master` image

do either of you remember what the different tags are supposed to correspond to?

Reply:
I cannot recall what K8S_RELEASE is used for, @Katharine might know since she added the variants.yaml?

Reply:
spelunking for context

K8S_RELEASE and having a matching version of kubectl was added by https://github.com/kubernetes/test-infra/pull/10851 to fix https://github.com/kubernetes/test-infra/issues/10850

when it was added, this was prior to release-1.14 being cut: https://github.com/kubernetes/test-infra/blob/7f748595c321764889c7ceed25c2423677a94cc8/images/kubekins-e2e/Makefile
- master -> stable
- experimental -> stable
- 1.13 -> stable-1.13
- 1.y -> stable-1.y

when release-1.14 was cut: https://github.com/kubernetes/test-infra/blob/f384d448dbf691f9bb1c83e83e750061b5a1cc9f/images/kubekins-e2e/Makefile
- master -> stable
- experimental -> stable
- 1.14 -> latest
- 1.y -> stable-1.y

when release-1.15 was cut: https://github.com/kubernetes/test-infra/blob/9c33ddf227184e44c7d27c5802a2cb5add710249/images/kubekins-e2e/variants.yaml
- master -> stable
- experimental -> stable
- 1.15 -> stable
- 1.14 -> latest (seems like this should have been moved to stable-1.14)
- 1.y -> stable-1.y

Reply:
so yes, please leave `experimental` at `stable` so that it matches `master`'s `stable`

Reply:
> I'm wary of this change, I think the intent is for experimental to match master in all dimensions except BAZEL_VERSION

experimental is meant to be >= master in all dimensions.

Reply:
naming is hard ... we have the release branches which are one after the other, and the master branch ahead of that. experimental is the same image, but potentially ahead of the master branch on using newer tooling.